### DISCUSSION TITLE
Affichage Synapps en plein page ?

### DISCUSSION THREAD

**Discussion by Eltis74** [witsa/synapps#761] [Q&A] [answered]
Bonjour, 

J'aimerais savoir si il est possible d'afficher la Synapps se trouvant dans le redy en pleine page ?
Un peut comme pour les synoptiques de manière à ne pas avoir le bandeau Exploitation/Parametrage/Configuration.

Si oui pouvez vous me donner l'astuce ?

Merci 
CONTAT Arthur - Eltis

**Comment by WITNicolas** [comment, marked as answer]
Bonjour,

Dans l'Utilisateur, il suffit de mettre (dans les préférences / url par défaut) : 
/easy/Synapps/**LABELDELASYNAPP**

**Comment by Eltis74** [comment]
Super !!!
Merci du tuyau.
